.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# review when someone opens a pull request.
+# For more on how to customize the CODEOWNERS file - https://help.github.com/en/articles/about-code-owners
+*       @donnie-msft @dominoFire @dtivel @heng-liu @kartheekp-ms @nkolev92 @zivkan @zkat
```

Comment:
should we add @cristinamanum?

Reply:
Maybe, we can discuss that with her, when she's back. 

I added @dtivel cause he reviews PRs on a regular basis.